File: leetcode-java/src/test/java/com/github/anddd7/leetcode/SolutionTest.java
package com.github.anddd7.leetcode;

class SolutionTest {

  public static void main(String[] args) {
    System.out.println(
        Solution.INSTANCE.kthSmallest(
            new int[][]{
                new int[]{1, 5, 9},
                new int[]{10, 11, 13},
                new int[]{12, 13, 15}
            },
            8
        )
    );
  }
}

File: kotlin/leetcode-kotlin/src/test/kotlin/com/github/anddd7/leetcode/SolutionTest.kt
package com.github.anddd7.leetcode

import org.junit.Test

class SolutionTest {
  @Test
  fun test() {
    val listOf = listOf(0, 1, 2, 4, 2, 1)

    val printKMoves = Solution.findInMountainArray(
        3,
        object : Solution.MountainArray {
          override fun get(index: Int) = listOf[index]
          override fun length() = listOf.size
        }
    )
    println(printKMoves)
  }
}
